packages/core-mobile/app/navigation/AppNavigation.ts

```diff
@@ -199,7 +199,8 @@ enum ModalScreens {
   AnalyticsConsentSheet = 'ModalScreens.AnalyticsConsentSheet',
   UseWalletConnect = 'ModalScreens.UseWalletConnect',
   AlertScreen = 'ModalScreens.AlertScreen',
-  EnableNotificationsPrompt = 'ModalScreens.EnableNotificationsPrompt'
+  EnableNotificationsPrompt = 'ModalScreens.EnableNotificationsPrompt',
+  QRScanner = 'ModalScreens.QRScanner'
```

Comment:
The QRScanner, which was previously implemented as a modal, has been refactored into a React Navigation screen.